components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -132,7 +133,7 @@ private void printFailedRequest(FailedRequestException e) {
         }
     }
 
-    private String buildScanIndexToOrdinal(int i) {
+    private String runNumToOrdinal(int i) {
```

Comment:
Make static. Move to the end of this class.

Reply:
✅ Done